DQMOffline/Trigger/plugins/TopMonitor.cc

```diff
@@ -590,6 +600,22 @@ void TopMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup
   if (eventHT < HTcut_) return;
   if (MHTcut_>0 && eventMHT.pt()<MHTcut_) return;
 
+//george
+// put every muon in a vector
+  bool allpairs=false;
+  if (nmuons_>2) {
+    std::vector<TLorentzVector> mu;
+    for (unsigned int idx=0; idx<muons.size(); idx++){
+      TLorentzVector temp_mu;
+      temp_mu.SetPtEtaPhiM(muons[idx].pt(),muons[idx].eta(),muons[idx].phi(),0.105);
```

Comment:
please avoid the TLorentzVector conversion. in fact, its probably best just to inline the simple calculation in mllpairscuts altogether.